### PR TITLE
Add run label to Dockerfile and document usage

### PR DESCRIPTION
**Pull request by aweiteka**
Standard container LABELs have been documented[1]. Tools
such as the atomic CLI use these to identify how an image
is designed to be used. This simplifies the user experience
and provides a more reliable way to run a container.

[1] https://github.com/projectatomic/ContainerApplicationGenericLabels

### CODE REVIEW

**Comment by smarterclayton** [comment]
You'll need to add this to images/origin/Dockerfile as wel

**Comment by danmcp** [comment]
@aweiteka Needs rebase.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,29 @@ For questions or feedback, reach us on [IRC on #openshift-dev](https://botbot.me
 Getting Started
 ---------------
 
+The easiest way to run OpenShift Origin is in a Docker container (OpenShift requires Docker 1.6.2 or higher):
+
+**Important!**: Docker on non-RedHat distributions (Ubuntu, Debian, boot2docker) has mount propagation PRIVATE, which [breaks](https://github.com/openshift/origin/issues/3072) running OpenShift inside a container. Please use the [Vagrant](CONTRIBUTING.adoc#develop-on-virtual-machine-using-vagrant) or binary installation paths on those distributions.
+
+**Option 1**: Red Hat-based distributions (Fedora, CentOs, RHEL)
```
Comment (danmcp) [inline README.md]: CentOS case is off.

**Comment by danmcp** [comment]
@smarterclayton Bump

**Comment by danmcp** [comment]
@smarterclayton Bump

**Comment by aweiteka** [comment]
Due to the different use cases of the origin image I'm not sure we can target a single RUN cmd that would make sense. Closing.

**Comment by smarterclayton** [comment]
At this point oc cluster up is the intended way to launch openshift in a
container - there's too much variation for anything short of ansible or a
program to manage in the divergent environments.

**Pull request closed without merging**